uportal-war/src/main/resources/layout/theme/respondr/respondr.xsl

```diff
@@ -305,7 +305,7 @@
             return false;
           });
           // Console for debugging.
-          console.log("menu", menu, "menuToggle", menuToggle);
+          //console.log("menu", menu, "menuToggle", menuToggle);
```

Comment:
On an other side you could need some logs in development, in this case the best would be to provide a function that do logs and which is configured by a XSL property to activate logs.
But in all case I'm agree these lines should be cleaned !